traffic_monitor/todata/todata.go

```diff
@@ -222,20 +241,34 @@ func getDeliveryServiceServers(crc CRConfig) (map[tc.DeliveryServiceName][]tc.Ca
 		}
 
 	}
-	return dsServers, serverDses, nil
+	return dsServers, serverDses
 }
 
 // getDeliveryServiceRegexes gets the regexes of each delivery service, for the given CDN, from Traffic Ops.
 // Returns a map[deliveryService][]regex.
-func getDeliveryServiceRegexes(crc CRConfig) (Regexes, error) {
+func getDeliveryServiceRegexes(crc CRConfig, mc tc.TrafficMonitorConfigMap) (Regexes, error) {
 	dsRegexes := map[tc.DeliveryServiceName][]string{}
 
+	if canUseMonitorConfig(mc) {
+		for dsName, dsData := range mc.DeliveryService {
+			if len(dsData.HostRegexes) < 1 {
+				log.Warnln("TMConfig missing regex for delivery service " + string(dsName))
```

Comment:
Nit but as a `fmt.Stringer`, `DeliveryServiceName` values can be coerced automatically to strings - all you have to do is get rid of the manual cast and exchange ` " + ` for `, ` i.e.
```go
log.Warnln("TMConfig missing regex for delivery service " + string(dsName))
```
is equivalent to
```go
log.Warnln("TMConfig missing regex for Delivery Service", dsName)
```

Reply:
> Inserting a space between arguments is kind of surprising IMO.

I don't disagree, but it's Go's standard library doing it, technically, so not something we avoid without jumping through annoying hoops.